pages/_app.js

```diff
@@ -2,7 +2,6 @@ import React from 'react'
 import * as Sentry from '@sentry/browser'
 import App from 'next/app'
 import Head from 'next/head'
-import { Router } from 'components/Router'
 import Navbar from 'components/Navbar'
 import { Web3ConnectProvider } from 'lib/web3-connect'
 import env from 'lib/environment'
```

Comment:
Remember to update the sentry DSN here to be `bonding.aragon.org`.

We will also need to set up a new Sentry DSN for this app.